yorkie/backend/db/mongo/client.go

```diff
@@ -573,3 +580,8 @@ func (c *Client) collection(
 		Database(c.config.YorkieDatabase).
 		Collection(name, opts...)
 }
+
+func (c *Client) observePushpullSnapshotDurationSeconds(timer *util.Timer) {
```

Comment:
I think this is one of the premature abstractions. Subtracting this function doesn't seem to be of many benefits to us. It would be nice to express it inline again.